.pre-commit-config.yaml

```
@@ -8,7 +8,7 @@ repos:
       - id: licenseheaders
         args: ["--tmpl=LICENSE_SHORT", "--ext=py", "-f"]
   - repo: https://github.com/ambv/black
```

nit:
```suggestion
  - repo: https://github.com/psf/black
```

Looks like there's a redirection to a newest repo. (maybe the username changed)

---

oh good spot